backend/src/typeorm/entity/Balance.ts

```diff
@@ -0,0 +1,25 @@
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column } from 'typeorm'
+
+@Entity('state_balances')
+export class Balance extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  id: number
+
+  @Column({ name: 'state_user_id' })
+  userId: number
+
+  @Column({ type: 'datetime' })
+  modified: Date
+
+  @Column({ name: 'record_date', type: 'datetime' })
+  recordDate: Date
+
+  @Column({ type: 'bigint' })
+  amount: number
+
+  static findByUser(userId: number): Promise<Balance> {
+    return this.createQueryBuilder('balance')
+      .where('balance.userId = :userId', { userId })
+      .getOneOrFail()
```

Comment:
Please do not define queries in the model. I think we should separate queries and models. You can put the query in its own file if you intend to reuse it. eg. `queries/balance/findByUser.ts`

Reply:
So you rather like to use the Data Mapper Pattern over the Active Record pattern?
https://typeorm.io/#/active-record-data-mapper

Reply:
Or do you like it more if I go for your proposal and ignore the typeorm standards?

Reply:
I tried to find this, but couldnt when i scrolled over the docu of TypeORM, thanks ❤️ !

I think its a thing we could discuss with @Mogge and @Elweyn .
The documentation itself gives us a hint aswell - maintainability vs simplicity. We just have to decide what we want. It might be wise to decide this now to prevent larger refactors later.

![image](https://user-images.githubusercontent.com/1238238/134165705-d8e0a25b-88e8-4e9c-a093-44225514e8a3.png)

And ofc we will use the standard, I just could not find it in the docu!